forte/ontology_specs/clinical.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "clinical",
+  "imports": [
+    "forte/ontology_specs/base_ontology.json"
```

Comment:
I think the import like the following would work:
`
 "imports": [
    "base_ontology.json"
  ],
`

Reply:
Yes, fixed.